Source/Run_Methods.sql

```diff
@@ -735,7 +739,14 @@ BEGIN
        FOR XML EXPLICIT
        );
 
-    EXEC tSQLt.Private_PrintXML @XmlOutput;
+    If @CastToNvarchar = 1

```

Comment:
Rather than adding IF statement there, it would probably better be moved to the `Private_PrintXML` procedure as now outputting logic is split in 2 different places that kind of bad practice.
Additionally this framework is written using TDD approach so you should write tests to cover this functionality otherwise I'm not sure it will be merged.
And final minor comment is that keywords in this source code are all in UPPERCASE (If, nvarchar, max)

Reply:
Will do. Is there a CI framework that runs the tests?

Reply:
Currently you can only do that locally. If you go to the `Build` folder, you'll find `Install the tSQLt build.docx` document with instructions. And if you have troubles with the certificate, you can take a look to the `ChangeSigningKey_ReadMe.txt` document.
The tests should probably need to be put to the `Tests/Run_Methods_Tests.class.sql` file

Reply:
BTW, you won't be able to do that on Linux ... However as I see from commits, that @mbt1 is currently working on setting automated tests on GitHub so there might be such possibility in the near future.